lib/image.js

```diff
@@ -46,20 +43,22 @@ Object.defineProperty(Image.prototype, 'src', {
             throw err
           }
         }
-
-        if (!get) get = require('simple-get')
-
-        get.concat({
-          url: val,
+        const request =  fetch(url = val, data = {
+          method: "POST",
           headers: { 'User-Agent': 'Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/102.0.0.0 Safari/537.36' }
-        }, (err, res, data) => {
-          if (err) return onerror(err)
+        })
 
+        request.then((res) => {
```

Comment:
Since `request` is only used once, you could chain here and remove the variable:

```suggestion
        })
        .then((res) => {
```

Reply:
=> https://github.com/Automattic/node-canvas/pull/2309/commits/93cfd2f012cf40bbf0d7a7f88f60b9c3659106a2